docs/source/kedro_project_setup/settings.md

```diff
@@ -0,0 +1,31 @@
+# Project settings
+
+A Kedro project's `settings.py` file contains the runtime configuration for the project, including registration of hooks and library components. This page explains how settings work and which settings are available.
+
+By default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
+
+## Available settings
+
+```eval_rst
++-----------------------------------+---------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Setting                           | Default value                                           | Use                                                                                                                                                     |
++===================================+=========================================================+=========================================================================================================================================================+
+| :code:`HOOKS`                     | :code:`tuple()`                                         | Instantiate and list your `project hooks <../extend_kedro/hooks.md>`_.                                                                                  |
```

Comment:
```suggestion
| :code:`HOOKS`                     | :code:`tuple()`                                         | Instantiate and list your project hooks <../extend_kedro/hooks.md>`_.                                                                                  |
```

Reply:
The ` is needed to render this piece of text as a link.
<img width="300" alt="Screenshot 2021-12-07 at 12 26 12" src="https://user-images.githubusercontent.com/49397448/145028802-be515f85-a6c2-49c8-a9a0-1cd97868e989.png">